scripts/notes.ts

```diff
@@ -150,12 +150,14 @@ const organizeNotes = (data: PRDetails[]) => {
   // initial sort
   notes.sort();
 
-  // move cmsgov to end
-  notes.forEach((note: PRNote) => {
-    if (note[0] === 'cmsgov') {
-      notes.push(note);
-      notes.shift();
-    }
+  // move cmsgov to end of list
```

Comment:
Regarding lodash, I don't care if we use lodash for tooling. It's not like it'd be adding more dependencies to the component library